dags/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
A trick I learned a couple months ago: you can do the same using:
`find . -name .directory -delete`